test/routes/tsoa/authorize.spec.ts

```diff
@@ -238,6 +239,76 @@ describe("authorize", () => {
       expect(actual).toBe("Redirect to login");
       expect(controller.getStatus()).toBe(302);
     });
+
+    it("should use audience from the request for the silent auth", async () => {
+      // https://auth2.sesamy.dev/authorize
+      //     ? client_id = VQy2yYCA9rIBJerZrUN0T
+      //     & scope=openid+profile+email
+      //     & redirect_uri=http://localhost:3000
+      //     & prompt=none
+      //     & response_type=token id_token
+      //     & response_mode=web_message
+      //     & state=state
+      //     & nonce=cUdmMWo1eFgubzdjMU9xSmhiS0pYdmpJME1GbFpJcllyWnBTU1FnWXQzTA % 3D % 3D
+      //     & code_challenge=CA6jwqDHtqZIzs9dcmTNBavFDQHPkfuBIO2Q8XRvWGA
+      //     & code_challenge_method=S256
+      //     & auth0Client=eyJuYW1lIjoiYXV0aDAtcmVhY3QiLCJ2ZXJzaW9uIjoiMi4xLjAifQ % 3D % 3D
+      const controller = new AuthorizeController();
+
+      const stateData: { [key: string]: any } = {
+        // This id corresponds to the base64 token below
+        c20e9b02adc8f69944f036aeff415335c63ede250696a606ae73c5d4db016217:
+          JSON.stringify({
+            userId: "tenantId|test@example.com",
+            authParams: {
+              redirect_uri: "https://example.com",
+              scope: "openid profile email",
+              state:
+                "Rk1BbzJYSEFEVU9fTGd4cGdidGh0OHJnRHIwWTFrWFdOYlNySDMuU3YxMw==",
+              client_id: "clientId",
+              audidence: "",
```

Comment:
```suggestion
              audience: "",
```
:exclamation: 

This needs changing this could be a false positive :smile: 

Can we type this properly so this is caught?

Reply:
I'll commit this @markusahlstrand I want to see what happens :smile: